### ISSUE TITLE
Shorts still appear in search results.

### ISSUE THREAD

**Issue by SteegHobland** [doma-itachi/Youtube-shorts-block#36]
I'm using Firefox, and installing this extension didn't noticeably change how search result displayed. There's still panels of shorts that I don't want to see.

**Comment by doma-itachi** [comment]
You should enable "Hide shorts video".
![あ](https://user-images.githubusercontent.com/94786908/226177687-ad997ddb-af64-4384-8fe5-afbda9117d61.png)
![あ](https://user-images.githubusercontent.com/94786908/226177689-c83a91da-926f-4a61-88fa-875ffe881123.png)

**Comment by SteegHobland** [comment]
Yes, I tried checking the **hide "shorts" tab** option, and both that and the **hide shorts video(unstable)** option. No apparent change in what I see in search results. There are still some rows of shorts displayed.

**Comment by ToxicMushroom** [comment]
![image](https://github.com/doma-itachi/Youtube-shorts-block/assets/32853531/7ceb21cf-18a1-4ca8-981a-718bc11e8a1c)
I also have it show up now, I think you could block `ytd-reel-shelf-renderer` tags as an easy fix ?

**Comment by doma-itachi** [comment]
@ToxicMushroom
Hmmm... The reel is blocked if "hide shorts video" is enabled in the settings. Is it enabled?
If it is already enabled and the reel is still visible, it is probably a bug in the extension.
![image](https://github.com/doma-itachi/Youtube-shorts-block/assets/94786908/90552e99-9095-41bb-a7e8-cf73abd5516a)

![image](https://github.com/doma-itachi/Youtube-shorts-block/assets/94786908/9b56dbcf-cd9a-4cd5-a3b5-8aa6308b2529)

**Comment by ToxicMushroom** [comment]
yes those options were on but I can't reproduce it anymore today :thinking:
